app/validation.rb

```diff
@@ -0,0 +1,9 @@
+module Validation
+  # def validtion_name(name)
+  #   name.is_a?(String) && !name.empty? && name.length.between?(3, 20)
+  # end
+
+  def validtion_difficulty(difficulty)
```

Comment:
1) why do you use module with 1 method
2) please - don't create validations FOR separate ATTRIBUTES - create just for small check of something